spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -234,17 +236,29 @@ def _view; end # Stub method
         expect(view_spec.view).to eq(view)
       end
 
-      it 'is accessible to hooks' do
+      if RUBY_VERSION <= "2.3.0" && ENV["RAILS_VERSION"] !~ /stable/ && ::Rails.version.to_f == 5.2
+        pending_only_on_ruby_22_rails_52 = """
+          Rails 5.2.4.2 has a syntax error in ActionDispatch::Request::Session.
+          (A &. usage which does not work in 2.2.10)
+          It has been fixed but not released, this spec will not pass until that
+          has been released.
+        """
+      else
+        pending_only_on_ruby_22_rails_52 = false
+      end
+
+      it 'is accessible to hooks', pending: pending_only_on_ruby_22_rails_52 do
         with_isolated_config do
           run_count = 0
           RSpec.configuration.before(:each, type: :view) do
             allow(view).to receive(:render) { :value }
             run_count += 1
           end
           group = RSpec::Core::ExampleGroup.describe 'a view', type: :view do
-            specify { true }
+            specify { expect(true).to eq true }
           end
-          group.run
+          group.run(failure_reporter)
+          expect(failure_reporter.exceptions).to eq []
```

Comment:
I do not understand why we need the `FailureReporter` anymore?

Reply:
Its included for future debugging, we've had this spec fail before and its really hard to see why without this pattern, I'm going to use it in other places for the same reason.

Reply:
Nice! Maybe we could add a little bit of documentation on the top of file or somewhere?